### ISSUE TITLE
About the unstable performance

### ISSUE THREAD

**Issue by tjulyz** [HKUST-Aerial-Robotics/Stereo-RCNN#41]
Hi,
Thanks for your kind share. When I try to run the train step several times to get several models, I found that the performance are unstable. Sometimes, the AP3D for hardset may degrade by 3%.  I have got results (~51.1, ~34.5, ~30.1 for easy, moderate, hard). Could you please help me? 
Thanks a lot!

**Comment by PeiliangLi** [comment]
Hi, it is the best model you trained or the worst model?

**Comment by tjulyz** [comment]
Hi, it is the worst model maybe. But I am not sure. I have got about 27％ AP for hard set on another computer. So is the performance reported by average or by model selection？Thanks a lot!

**Comment by PeiliangLi** [comment]
Usually, ~1% variance should be normal due to the random property. But I didn't see such a different AP. Please make sure using the exact parameters as we provide.

**Comment by tjulyz** [comment]
Thanks. I train the model with your provided parameters. For the easy set，the variance is much larger than other sets.

**Comment by tjulyz** [comment]
Hi, I have another question about the test set performance. Did you directly test the model which is trained on training set(3000+images) and reported the test set(7500+images) performance? Have you merged the training set and validation set to retrain the model when evaluated on the test set?
Thanks a lot!
